script/Config/App.lua

```diff
@@ -1,10 +1,14 @@
 Config.app = 'LTheoryRedux'
 
+Config.version = "v0.004"
```

Comment:
might be good using semver

Reply:
I don't think we should be committing a version directly to the code either. Ideally, a script that generates a redistributable package should set the version number itself.

Reply:
I'm open to automated approaches that the code itself can access for display. This is just a temporary mechanic.

Reply:
> I don't think we should be committing a version directly to the code either. Ideally, a script that generates a redistributable package should set the version number itself.

since we use GH Actions for building we can just set up a string tag in the code and replace that via the workflow.